### DISCUSSION TITLE
Procedure for making a new panel based on a saved copy of another?

### DISCUSSION THREAD

**Discussion by Andy2No** [RomanKubiak/ctrlr#633] [Q&A] [answered]
What's the best way to make a new panel based on another one, i.e by saving a copy then editing the copy?  I seem to remember there's some problem with the ID staying the same, but I can't find that information.

What needs to be changed, in the copy, to give it its own unique existence, and what's the best way to make the changes - e.g. within Ctrlr, or by editing the .panel file with a text editor?

I have a panel I used to use a remote control for my Yamaha TX802, because it wasn't in reach (mostly by someone else, with some changes by me).  Although they're quite different, I'd like to use that as a basis for one for my Yamaha EX5R.  I could adapt the remote sysex control of buttons part quite quickly, which would be a good enough start.

**Comment by dobo365** [comment, marked as answer]
Hi,
  it's rather easy in fact. Always once you know it ;-) 

- Take a backup copy of your existing panel (optional)
- Open your existing panel
- Select **File - Export - Regenerate ID** (look first to the 4 last digits to identify the old ID, it will be needed later)
- When doing that you will get the question "Are you sure? The resource directory has changed, do you want to delete the old one?
You should say **No**.
We will change this question has it is misleading.
In fact, the ID has already been changed, a new temp directory has been created using the new ID. The question relates to the fact if the old resources (images, font files, data files...) should be kept or not
- Do **File - Save As** and save your panel with a new name
- Do **File - Save** to verify in the green warning that the new name is OK
- Close the panel
- In Windows Explorer or Finder, go to the temp folder of your existing panel. On Windows, it is under Users/AppData/Roaming/Ctrlr/_old_panel_ID_
- Select all images and press Copy
- Paste all images to the folder Users/AppData/Roaming/Ctrlr/_new_panel_ID_
- Open your new panel in Ctrlr and it will be like the old one

**Reply by Andy2No** [reply]
Thanks for the detailed instruction, @dobo365
